sorc/checkout.sh

```diff
@@ -131,7 +131,8 @@ mkdir -p ${logdir}
 # The checkout version should always be a speciifc commit (hash or tag), not a branch
 errs=0
 checkout "ufs_model.fd"    "https://github.com/ufs-community/ufs-weather-model" "${ufs_model_hash:-Prototype-P8c}"; errs=$((errs + $?))
-checkout "gsi.fd"          "https://github.com/NOAA-EMC/GSI.git"                "a62dec6"                         ; errs=$((errs + $?))
+checkout "gsi.fd"          "https://github.com/NOAA-EMC/GSI.git"                "develop"                         ; errs=$((errs + $?))
```

Comment:
The current head of NOAA-EMC/GSI develop is [c10644e](https://github.com/NOAA-EMC/GSI/commit/c10644ee443cb7a99c4300b9037d705a49501023).   My working copy of `sorc/checkout.sh` now specifies this hash instead of develop.   I'll update this hash again if NOAA-EMC/GSI develop is updated before this PR is merged into g-w develop.